### PR TITLE
Link PPV report to PP FAQ for valid HTML version

### PR DESCRIPTION
**Pull request by srjfoo**
GM request: Update PPV report card form and email body to link to/refer to PP FAQ for valid HTML versions. Also harmonized a couple of line items in the email body with their corresponding item in the form. Testable in the [revise-ppv-form](https://www.pgdp.org/~srjfoo/c.branch/revise-ppv-form/) sandbox.

### CODE REVIEW

**Review by cpeel — APPROVED** [review]

**Review by 70ray — APPROVED** [review]

**Review by chrismiceli — APPROVED** [review]

File: tools/post_proofers/ppv_report.php [hunk 1]
```diff
@@ -512,7 +512,7 @@ function is_decimal_digits($s)
                 . check_box('s_code', _("Keep your code line lengths reasonable"))
                 . check_box('s_tables', _("Tables should display left, right, and center justification and top and bottom align appropriately"))
                 . check_box('s_th', _("Tables contain &lt;th&gt; elements for headings"))
-                . check_box('s_thumbs', _("Remove thumbs.db file from the images folder"))
+                . check_box('s_thumbs', _("Remove unwanted system/hidden OS files, e.g. thumbs.db or .DS_Store, from the images folder"))
```
[the remark this answers is not in the document]
Comment (srjfoo) [reply tools/post_proofers/ppv_report.php:515]: Thanks for spotting that -- I need to get that fixed.

[the remark this answers is not in the document]
Comment (srjfoo) [reply tools/post_proofers/ppv_report.php:515]: @70ray -- can you give me a line number? I've checked, and only see two lines where `thumbs` is mentioned, and both of them also now give `.DS_Store` as an example. LInes 515 and 788

[the remark this answers is not in the document]
Comment (srjfoo) [reply tools/post_proofers/ppv_report.php]: Thanks, got it now. That's what I get for not doing a case-insensitive check. And yes, definitely relevant.

**Comment by lhamilton1** [comment]
I approve.

**Pull request merged**